stubs/requests/requests/sessions.pyi

```diff
@@ -67,12 +67,14 @@ _Params: TypeAlias = Union[
     str | bytes,
 ]
 _TextMapping: TypeAlias = MutableMapping[str, str]
+_HeadersMapping = TypeAlias = MutableMapping[str, str | bytes]
```

Comment:
This is inconvenient because MutableMapping is invariant in its value type, so a `dict[str, str]` would not be acceptable.

Reply:
What about 
```
MutableMapping[str, str] | MutableMapping[str, bytes]
```
?

Reply:
0e8ff7e